ex2: 

Homework

ex3: 

# Introduction

ex4: 
# the list of data science languages
1. python
2. R
3. Scala

ex5: 

# the list of data science libraries 
1. scikit-learn
2. tensorflow
3. pytorch

ex6:

| Data science tool name |
| --- 
| Apache Spark |
| Tableau |
| RapidMiner |

ex7:

$$+$$ is for summation
$$-$$ is for subtraction
$$/$$ is for division
$$*$$ is for multiplying

In [5]:
# ex8

3*5 + 2

17

ex9: 7654273 minutes is 

In [6]:
7654273/60

127571.21666666666

equals to 127571.21666666666 hours

ex10:

# list of objectives
<ol>
<li>objective 1</li>
<li>objective 2</li>
<li>objective 3</li>
</ol>

ex11:

Author name: Onur Çatmabacak